app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -60,7 +60,7 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
       label: yup.string().oneOf(CONTACTS.map(contact => contact.label)),
       value: yup.string().oneOf(CONTACTS.map(contact => contact.value)),
     }),
-  tag: yup
+  tags: yup
     .object()
     .nullable()
     .shape({
```

Comment:
This validation is still not right. How does it account for multiple tags?

Reply:
I have corrected this. Thanks for the review.